app/controllers/home_controller.rb

```diff
@@ -1,7 +1,10 @@
 # frozen_string_literal: true
 
 class HomeController < ApplicationController
-  before_action :authenticate_user!
+  protect_from_forgery except: :piwik
+  skip_before_action :store_current_location, only: :piwik
+  # rubocop sees that as a hash ?!
+  before_action :authenticate_user!, :except => [:piwik] # rubocop:disable Style/HashSyntax
```

Comment:
Thanks for the tip !

I'm quite new or rusty in RoR/ruby pick one ;)

Correction pushed tomorrow !